lib/build.gradle

```diff
@@ -30,33 +29,13 @@ oss {
     }
 }
 
-def javaVersion = 8
-java {
-    toolchain {
-        languageVersion = JavaLanguageVersion.of(11)
```

Comment:
Doesn't seem to be needed even when the library outputs a multi-jar with module support. 

![image](https://user-images.githubusercontent.com/3900123/123821615-da8d0d00-d8fb-11eb-8f43-f34ccb5e893a.png)

Reply:
@jimmyjames can you please double-check this one on your side? for me it keeps compiling after running something like `./gradlew clean build javadoc publishMavenJavaPublicationToMavenLocal --rerun-tasks`

Reply:
Yes, if you are running gradle with java 11 it will work. Using a toolchain you should be able to run gradle with say, java 8, and gradle will use the specified toolchain version for tasks. However the custom `compileModuleInfoJava` task does not appear to be respecting the toolchain, instead using the java version running gradle. Since our CI runs on 11 and this issue exists outside this change, I'd at least add a comment to the custom task with a todo to use the toolchain version so we can run the build with a different java version.

Reply:
![image](https://user-images.githubusercontent.com/3900123/124135697-587c2000-da84-11eb-8200-5fcaa192537c.png)

I tried changing my local JDK back to JDK 8 and ran the command above. The build is still successful. Do you think the comment is necessary? And, would that be for the "compileJava" task?

Reply:
Per our discussion today, I'll add this block back to prevent potential issues when running the build using JDK 8.